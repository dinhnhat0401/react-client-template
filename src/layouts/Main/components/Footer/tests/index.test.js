import React from 'react';
import { render } from '@testing-library/react';

import Footer from '../';

describe('<Footer />', () => {
  it('should render and match the snapshot', () => {
    const tree = render(<Footer />);
    expect(tree).toMatchSnapshot();
  });
});
