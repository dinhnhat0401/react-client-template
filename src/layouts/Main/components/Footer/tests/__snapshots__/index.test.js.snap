// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render and match the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="makeStyles-root-1"
      >
        <p
          class="MuiTypography-root MuiTypography-body1"
        >
          ©
           
          <a
            class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
            href="https://devias.io/"
            target="_blank"
          >
            Devias IO
          </a>
          . 2019
        </p>
        <span
          class="MuiTypography-root MuiTypography-caption"
        >
          Created with love for the environment. By designers and developers who love to work together in offices!
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="makeStyles-root-1"
    >
      <p
        class="MuiTypography-root MuiTypography-body1"
      >
        ©
         
        <a
          class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
          href="https://devias.io/"
          target="_blank"
        >
          Devias IO
        </a>
        . 2019
      </p>
      <span
        class="MuiTypography-root MuiTypography-caption"
      >
        Created with love for the environment. By designers and developers who love to work together in offices!
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
